src/test/java/org/jenkins/tools/test/PluginCompatTesterTest.java

```diff
@@ -216,30 +216,6 @@ public void testWithCasCProperties() throws Throwable {
         tester.testPlugins();
     }
     
-    @Test
-    public void testWithInvalidExclusionList() throws Throwable {
-        File exclusionList = new ClassPathResource("bad-surefire-exclusion-list").getFile();
-        PluginCompatTesterConfig config = getConfig(ImmutableList.of("active-directory"));
-        Map<String, String> mavenProperties = new HashMap<>();
-        mavenProperties.put("surefire.excludesFile", exclusionList.getAbsolutePath());
-        config.setMavenProperties(mavenProperties);
-        
-        PluginCompatTester tester = new PluginCompatTester(config);
-        PluginCompatReport report = tester.testPlugins();
-        assertNotNull(report);
-        Map<PluginInfos, List<PluginCompatResult>> pluginCompatTests = report.getPluginCompatTests();
-        assertNotNull(pluginCompatTests);
-        for (Entry<PluginInfos, List<PluginCompatResult>> entry : pluginCompatTests.entrySet()) {
-            assertEquals("active-directory", entry.getKey().pluginName);
-            List<PluginCompatResult> results = entry.getValue();
-            assertEquals(1, results.size());
-            PluginCompatResult result = results.get(0);
-            assertNotNull(result);
-            assertNotNull(result.status);
-            assertEquals(TestStatus.INTERNAL_ERROR, result.status);
-        }
-    }
-
```

Comment:
The point of this was to test that an exclusion list with `io.jenkins.plugins.FooBar#method` resulted in a status of `INTERNAL_ERROR`. As of recent changes to the Surefire plugin, that is actually supported and _should not_ have caused an error, but the test was passing erroneously for unrelated reasons. Since this use case is no longer valid, deleting the test.